Backend/DataAccessLayer/Migrations/201903051936518_PasswordReset.cs

```diff
@@ -8,27 +8,27 @@ public partial class PasswordReset : DbMigration
         public override void Up()
         {
             CreateTable(
-                "dbo.ResetIDs",
+                "dbo.PasswordResets",
                 c => new
                 {
-                    PasswordResetID = c.Guid(nullable: false),
+                    Id = c.Guid(nullable: false),
                     ResetToken = c.String(nullable: false),
                     UserID = c.Guid(nullable: false),
                     ExpirationTime = c.DateTime(nullable: false, precision: 7, storeType: "datetime2"),
                     ResetCount = c.Int(nullable: false),
                     Disabled = c.Boolean(nullable: false),
                     AllowPasswordReset = c.Boolean(nullable: false),
                 })
-                .PrimaryKey(t => t.PasswordResetID)
+                .PrimaryKey(t => t.Id)
                 .ForeignKey("dbo.Users", t => t.UserID, cascadeDelete: true)
                 .Index(t => t.UserID);
         }
         
         public override void Down()
         {
-            DropIndex("dbo.ResetIDs", new[] { "UserId" });
-            DropForeignKey("dbo.ResetIDs", "UserId", "dbo.Users");
-            DropTable("dbo.ResetIDs");
+            DropIndex("dbo.PasswordResets", new[] { "UserId" });
+            DropForeignKey("dbo.PasswordResets", "UserId", "dbo.Users");
+            DropTable("dbo.PasswordResets");
         }
```

Comment:
Again, this migration really can't be modified. Create a new migration that drops and re-creates this table with the correct format, or find a way to migrate the column names.

Reply:
The problem is that I want to change the name of the table/columns in both the code and the database. 

I can't update the column names because it can't find a dbo.PasswordResetIDs table. If I change the Dbset line back to PasswordResetIDs, then the repository code that references a table called PasswordReset fails to build.

I'm stuck between two choosing two options that both won't work.